scripts/globals/atma.lua

```diff
@@ -163,7 +163,7 @@ function atmaEffectGain(target, effect)
     elseif (pwr == 43) then -- dsp.ki.ATMA_OF_THE_DEEP_DEVOURER
         target:addMod(dsp.mod.SUBTLE_BLOW, 5);
         target:addMod(dsp.mod.STORETP, 5);
-        -- target:addMod(MOD_SONG_CASTING_TIME, -20); Doesnt exist
+        -- target:addMod(dsp.mod.SONG_CASTING_TIME, -20); Doesn't exist
```

Comment:
this should be SONG_SPELLCASTING_TIME (mod 455, according to modifier.h) and it's missing from status.lua